### ISSUE TITLE
kwargs.request is being set when there is no request

### ISSUE THREAD

**Issue by robinjoseph08** [getsentry/raven-node#321]
When using v2.0.1 on a non-web application (i.e. a worker process), the client is setting `kwargs.request` with garbage information which then clutters up the issue page when viewing the exception:

```js
  request:
   { method: undefined,
     query_string: {},
     headers: {},
     cookies: {},
     data: '<unavailable>',
     url: 'http://<no host>undefined' },
```

![image](https://cloud.githubusercontent.com/assets/2454505/26288111/795da068-3e3f-11e7-9acd-4523afc1de63.png)

Ideally, this isn't populated at all since there's no relevant "request" within the context of this worker.

From preliminary digging, it looks like it's because `parsers.parseRequest` adds this info even if it doesn't exist.

**Comment by LewisJEllis** [comment]
Thanks for the report, @robinjoseph08; I noticed this but thought it was just something strange I was doing in a testing case, didn't realize it was a general problem!

I opened #321 to fix, after it merges I'll publish 2.0.2.

**Comment by LewisJEllis** [comment]
Published in 2.0.2.

**Comment by robinjoseph08** [comment]
Thanks a ton for the quick turnaround!
